Tests/Python/PythonOptionTests.cs

```diff
@@ -107,61 +107,5 @@ public void FilterReturnsUniverseRegression()
                 parameter.Language,
                 parameter.ExpectedFinalStatus);
         }
-
-        [Test]
-        public void FilterReturnsListRegression()
```

Comment:
Removed because this regression `BasicTemplateOptionsFilterUniverseAlgorithm` already has a C# implementation that runs the algo in `AlgorithmStatisticsRegression` so this is a duplicate really